pythia/common/defaults/configs/datasets/vqa/m4c_stvqa.yml

```diff
@@ -0,0 +1,22 @@
+dataset_attributes:
```

Comment:
Configs currently have a lot of replication and not fully utilizing the power of inheritance in our configuration system.